packages/@okta/vuepress-site/docs/reference/api/org/index.md

```diff
@@ -0,0 +1,857 @@
+
+---
+title: Org
+category: management
+---
+
+# Org API
+
+<ApiLifecycle access="ea" />
+
+The Okta Org API provides operations to manage your org account settings such as contact information, granting Okta Support access, and more.
+
+## Get Org Settings
+
+<ApiOperation method="get" url="/api/v1/org" />
+
+Gets your Organization's current settings.
+
+### Response parameters
+
+The [Org Object](#org-object)
+
+### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/org"
+```
+
+### Response example
+
+```json
+{
+    "id": "00ou8s5wploBwX4710g3",
+    "subdomain": "okta",
+    "name": "Okta",
+    "status": "ACTIVE",
+    "expiresAt": null,
+    "created": "2020-10-26T15:03:08.000Z",
+    "lastUpdated": "2021-01-20T21:02:28.000Z",
+    "website": "http://okta.com",
+    "phoneNumber": "+1-555-415-1337",
+    "endUserSupportHelpURL": "https://support.okta.com",
+    "supportPhoneNumber": "+1-555-514-1337",
+    "address1": "301 Brannan St.",
+    "address2": "Unit 100",
+    "city": "San Francisco",
+    "state": "California",
+    "country": "United States of America",
+    "postalCode": "94107",
+    "_links": {
+        "preferences": {
+            "href": "http://okta.okta.com/api/v1/org/preferences"
+        },
+        "uploadLogo": {
+            "href": "http://okta.okta.com/api/v1/org/logo",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        },
+        "oktaCommunication": {
+            "href": "http://okta.okta.com/api/v1/org/privacy/oktaCommunication"
+        },
+        "logo": {
+            "href": "http://okta.okta.com/bc/image/fileStoreRecord?id=fs02ju1ejvy2Cv2Yx0g4"
+        },
+        "oktaSupport": {
+            "href": "http://okta.okta.com/api/v1/org/privacy/oktaSupport"
+        },
+        "contacts": {
+            "href": "http://okta.okta.com/api/v1/org/contacts"
+        }
+    }
+}
+```
+## Update Org Settings
+
+> **Note:** Use the `POST` method to make a partial update and the `PUT` method to delete unspecified properties.
+
+<ApiOperation method="put" url="/api/v1/org" />
+
+Updates your Organization's current settings
+
+All org properties must be specified when updating an org's profile with a `PUT` method. Any property not specified in the request is deleted.
+
+>Important: Don't use `PUT` method for partial updates.
```

Comment:
```suggestion
> **Note:** Don't use `PUT` method for partial updates.
```

Reply:
just double checking, I copied this from https://developer.okta.com/docs/reference/api/users/

Reply:
The only callout we have is note. Important is not a thing right now. Unfortunately our API docs lack consistency more generally...